community-operators/rook/ceph_cr.yaml

```diff
@@ -0,0 +1,130 @@
+#################################################################################################################
```

Comment:
Please do not include CR objects with your operator manifests. The only objects needed for PRs are:
- ClusterServiceVersions
- CustomResourceDefinitions
- package.yaml

Reply:
You can include example CRs inside your CSV in the `metadata.annotations.alm-examples` section, which is a stringified json list of example CRs

Reply:
What about this test? https://github.com/operator-framework/community-operators/blob/master/scripts/ci/test-operator#L130